graph-builder/src/graph.rs

```diff
@@ -17,13 +17,34 @@ use cincinnati::{plugins, AbstractRelease, Graph, Release, CONTENT_TYPE};
 use commons::GraphError;
 use config;
 use failure::Error;
+use prometheus::{Counter, IntGauge};
 use registry::{self, Registry};
 use serde_json;
 use std::collections::{HashMap, HashSet};
 use std::sync::{Arc, RwLock};
 use std::thread;
 
+lazy_static! {
+    static ref GRAPH_DAG_NODES: IntGauge = register_int_gauge!(
+        "cincinnati_gb_graph_dag_nodes",
+        "Number of nodes (releases) in the graph."
+    )
+    .unwrap();
+    static ref GRAPH_FETCHED_RELEASES: IntGauge = register_int_gauge!(
+        "cincinnati_gb_graph_upstream_fetched_releases",
```

Comment:
With the above rename this could be `cincinnati_gb_graph_releases_upstream` or maybe `_raw`

Reply:
Current form matches the verb and help disambiguation: `fetch_releases()` -> `fetched_releases`. I can drop the "fetch" part if you prefer, but I think it makes more sense to rename the one above and keep this.

Reply:
I understand that motivation, but I think it's unnecessarily close to the names in the code. I was trying to find high-level names which are meaningful without knowing the details of the codebase.